src/patterns/gen-ai/aws-text-to-sql/README.md

```diff
@@ -0,0 +1,319 @@
+| :------------------------------ | :------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------ |
+| AWS Step Functions            | 100000 workflow request with maximum 15 state transition                                                                                        | 37.40                  |
+| Amazon EventBridge            | 1 million custom events with payload size of 10 kb                                                                                              | 1                      |
+| AWS Lambda                    | 100000 invocation request, x86 arch, 3 sec duration of each request, 7076 MB, ephermal storage 512 MB. ~~ 27.88 per lambda function . 4 X 27.88 | 111.52                 |
+| Amazon Simple Storage Service | 100000 GET request, 100 GB data transfer.                                                                                                       | 0.11                   |
+| Amazon Bedrock                | With the on-demand mode, for around 2000 request per day with 2000 input token and 1000 output token per request, with Anthropic Claude 3 Haiku | 95.00                  |
+| Amazon Aurora MySql           | With 1 Node , db.t3.medium size, 150 hours of usage per month(1 instance(s) x 0.082 USD hourly x 150 hours in a month)                          | 12.30                  |
+| Amazon CloudWatch             | 15 metrics using 5 GB data ingested for logs                                                                                                    | 7.02                   |
+| AWS X-Ray                     | 100,000 requests per month through AppSync and Lambda calls                                                                                     | 0.50                   |
+| Total Deployment cost         | For roughly around 40,000 requests(2000 input tokens / average request and 1000 output tokens/ average request)                                 | 264.85                 |
+
+The resources not created by this construct ( AWS Secrets Manager secret ) do not appear in the table above. You can refer to the decicated pages to get an estimate of the cost related to those services:
+
+- [AWS Secrets Manager Pricing](https://aws.amazon.com/secrets-manager/pricing/)
+
+## Security
+
+When you build systems on AWS infrastructure, security responsibilities are shared between you and AWS. This [shared responsibility](http://aws.amazon.com/compliance/shared-responsibility-model/) model reduces your operational burden because AWS operates, manages, and controls the components including the host operating system, virtualization layer, and physical security of the facilities in which the services operate. For more information about AWS security, visit [AWS Cloud Security](http://aws.amazon.com/security/).
```

Comment:
I think we need to add a note here about query execution. The construct might execute a query against the DB if configured to do so, so users need to be explicitly aware of that.

Reply:
d28bbe26726f810317c2b3533f33c248fd7c9cfb